sample-schemas/demo_schema.erl

```diff
@@ -0,0 +1,60 @@
+-module(demo_schema).
+
+-include_lib("typerefl/include/types.hrl").
+
+-type ip() :: ipv4() | ipv6().
+-type ipv4() :: {0..255, 0..255, 0..255, 0..255}.
+-type ipv6() :: {0..65535, 0..65535, 0..65535, 0..65535, 0..65535, 0..65535, 0..65535, 0..65535}.
+
+-reflect_type([ip/0]).
+
+-export([keys/0]).
+
+keys() ->
+    [ {"foo.setting", fun setting/1}
+    , {"foo.endpoint", fun endpoint/1}
+    , {"foo.greet", fun greet/1}
+    ].
+
+setting(mapping) -> "app_foo.setting";
+setting(type) -> string();
+setting(converter) -> converter(string);
```

Comment:
`converter` seems like an ambiguous term to me. Change to `to_string`?

Reply:
or `fun to_string/1`